### ISSUE TITLE
Admin: error and info icon is missing an alt text spoken to the screen reader

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#600]
## Notes
The alert error icon needs an alt text spoken to the screen reader user like <div role="img" aria-label="Error!"> so they hear "Error!" rather than just "Sorry ..."

Use aria-label="Info" for the info icon or something similar as the alt text. Use role=img to make it an image.  
<div aria-hidden="false" aria-live="assertive" class="usa-alert usa-alert-error" id="api-error" role="alert">
<div class="usa-alert-body">

<h3 class="usa-alert-heading">
Sorry, we were unable to process your request. Please try again.</h3>

<p class="usa-alert-text">Permit number 55555555 was not found.</p>

</div>
</div>
 
 
<div class="usa-alert usa-alert-info" id="season-updated-alert">
<div class="usa-alert-body">
<p class="usa-alert-text">Season dates for Mt. Hood National Forest are now Nov 01, 2018 to Dec 25, 2018</p>
</div>
</div>

![image](https://user-images.githubusercontent.com/28604890/37478290-310a105c-2850-11e8-9707-ceadc1b695d1.png)

![image](https://user-images.githubusercontent.com/28604890/37478304-370dae1e-2850-11e8-8991-55835cc9b44f.png)

Linkage: CTF-74
WCAG: 1.1.1

## Acceptance criteria
- [ ] error and other images in banners read "success" "alert" etc. before reading dialog (will)

## Tasks
- [ ] add aria-role="dialog" to all usa-alert-XXX (will)
- [ ] add aria-label="success" or "error" etc. to all usa-alert (will)

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by sadlerw** [comment]
update fs web design and if it isn't in the u.s. wds give them a PR

**Comment by sadlerw** [comment]
@JohnHewittCriterion there seems to be some confusion here as there are no images - they are in the css so it isn't possible to add alt text.  However I think changing the role from "alert" on the modal dialogs to "dialog" and adding a aria-label="error" should cause it to read "error".  Otherwise I don't see how to make it work since there isn't really anything there to add the role="img" to as you suggest.

**Comment by JohnHewittCriterion** [comment]
@sadlerw I think that seems reasonable.  It will read properly as an "Error" or "Warning" dialog and then read the text.  Or add an area-label to the element holding the image of "error" or "warning" and make sure it's in the div with the role="alert".  It should then read the "Error" then the message in the box.  I haven't done an exact test like this, but I've done similar and it worked well.

**Comment by sadlerw** [comment]
yea I like your solution better (in osx anyway) - if you make it a dialog then you have to press ctrl/option/shift/downarrow just to read the dialog text - it doesn't do it automatically.

**Comment by sadlerw** [comment]
<img width="602" alt="screen shot 2018-03-22 at 2 21 04 pm" src="https://user-images.githubusercontent.com/2058839/37793313-4875103e-2ddc-11e8-92a1-92931bec8f1c.png">
